src/services/sessionMgr.js

```diff
@@ -617,6 +621,37 @@ const activateNextBlock = async ({ userId }) => {
     await cleanCache(shortname)
   }
 
+  // TODO: pubsub
+  const savedsession = user.runningSession.toObject()
+  let activeBlock
+  const arrayLength = savedsession.blocks.length
+  for (let i = 0; i < arrayLength; i += 1) {
+    if (savedsession.blocks[i].status === QUESTION_BLOCK_STATUS.ACTIVE) {
+      activeBlock = savedsession.blocks[i]
+    }
+  }
+
+  if (activeBlock != null) {
+    const instanceId = activeBlock.instances[0]
+    const questioninstance = await QuestionInstanceModel.findById(instanceId)
+    const question = await QuestionModel.findById(questioninstance.question)
+    const questionPublic = Object()
+    const questionInfo = question.versions[question.versions.length - 1]
+    questionPublic.id = instanceId
```

Comment:
As far as I can see, these can just be added directly like

`
questionPublic = {
  questionId: question._id,
  title: question.title....
}
`